src/cfi.rs

```diff
@@ -233,18 +387,23 @@ impl<'input, Endian> FrameDescriptionEntry<'input, Endian>
     #[allow(dead_code)]
```

Comment:
This is still dead code. I don't think it hurts to have it, but I don't think this will ever be used outside of the tests? Also, maybe that means the tests shouldn't be using it?